OmiseSDK/InternetBankingSourceChooserViewController.swift

```diff
@@ -66,10 +63,8 @@ class InternetBankingSourceChooserViewController: AdaptableStaticTableViewContro
         tableView.deselectRow(at: indexPath, animated: true)
         let bank = element(forUIIndexPath: indexPath)
         
-        if #available(iOSApplicationExtension 10.0, *) {
-            os_log("Internet Banking Chooser: %{private}@ was selected", log: uiLogObject, type: .info, bank.description)
-        }
-        
+        os_log("Internet Banking Chooser: %{private}@ was selected", log: uiLogObject, type: .info, bank.description)
+
         let oldAccessoryView = cell?.accessoryView
         #if swift(>=4.2)
```

Comment:
There are also a few checks for the swift version. We can probably also remove them since we have Swift 5.0 as a minimum requirement?

Reply:
Removed all of the checking cases